src/JobRunner.php

```diff
@@ -63,31 +63,23 @@ public function addJob($class, array $definition = array())
 			return;
 		}
 
+		// If we have a run_time and interval set, we will ignore the interval when checking if job can run.
+		if (!is_null($definition->getInterval()) && !is_null($definition->getRunTime()))
+		{
+			$this->logger->warning("Both run_time and interval are set for {$reflection->getShortName()} — " .
```

Comment:
Just curious, would it be worth calling `$definition->setInterval(null);` here, rather than short-circuiting in the distant `canJobRun()`?